Clicker/Views/Settings/SettingsLogOutButtonCell.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  SettingsLogOutButtonCell.swift
+//  Clicker
+//
+//  Created by Lucy Xu on 3/23/19.
+//  Copyright © 2018 CornellAppDev. All rights reserved.
+//
+
+import UIKit
+
+protocol LogoutButtonCellDelegate: class {
+    func logOutButtonCellWasTapped()
+}
+
+class SettingsLogOutButtonCell: UICollectionViewCell {
+    
+    // MARK: Data vars
+    weak var delegate: LogoutButtonCellDelegate?
+    
+    // MARK: Views and VC's
+    var logOutButton: UIButton!
+    var linkURL: URL!
+    
+    // MARK: Layout constants
+    let standardInset = 18
```

Comment:
can you call this `buttonOffset` or something? just so we know what this corresponds to

Reply:
Addressed in most recent pull request!